app.js

```diff
@@ -0,0 +1,264 @@
+const bodyParser = require('body-parser')
+const cookieParser = require('cookie-parser')
+const express = require('express')
+const app = express()
+const exphbs = require('express-handlebars')
+const session = require('express-session')
+const MongoStore = require('connect-mongo')(session)
+const expressValidator = require('express-validator')
+const methodOverride = require('method-override')
+const mongoose = require('mongoose')
+const path = require('path')
+const server = require('http').createServer(app)
+
+const io = require('socket.io')(server)
+const flash =require('connect-flash')
+const passport = require('passport')
+const port = process.env.PORT || 3000;
+
+const routes = require('./routes/routes')
+const dbConfig = require('./config/dbConfig')
+const Twit = require('twit')
+const tweet = require('./helpers/twitter')
```

Comment:
i think you should group all of your helper files under `./helpers/` directory together